package/ports_daemon/ports_daemon/src/can.c

```diff
@@ -215,32 +219,34 @@ int can_init(settings_ctx_t *settings_ctx)
                     can_notify,
                     &cans[0]);
 
-  settings_register(settings_ctx,
-                    "can1",
-                    "bitrate",
-                    &cans[1].bitrate,
-                    sizeof(cans[1].bitrate),
-                    settings_type_bitrate,
-                    bitrate_notify,
-                    &cans[1]);
-
-  settings_register(settings_ctx,
-                    "can1",
-                    "tx_id",
-                    &cans[1].id,
-                    sizeof(cans[1].id),
-                    SETTINGS_TYPE_INT,
-                    can_notify,
-                    &cans[1]);
-
-  settings_register(settings_ctx,
-                    "can1",
-                    "rx_id_filter",
-                    &cans[1].filter,
-                    sizeof(cans[1].filter),
-                    SETTINGS_TYPE_INT,
-                    can_notify,
-                    &cans[1]);
+  if (is_duro()) {
```

Comment:
this looks wrong, remove parens?

Reply:
Another suggestion: remove the global bool and just call `device_is_duro` directly since this is only run at startup (which should be once per power cycle).